MIGRATION_GUIDE.md

```diff
@@ -6,6 +6,83 @@ across different versions.
 
 > [!TIP]
 > We highly recommend upgrading the versions one by one instead of bulk upgrades.
+ 
+## v0.98.0 ➞ v0.99.0
+
+### snowflake_task resource changes
+
+Breaking changes:
+- `config` field now can be defined without $ signs
```

Comment:
question: `can` or `must`? I mean when users pass with `$`, it won't work, right?

Reply:
I didn't notice, but this field was previously not available in the resource, so I'm going to just change it as `new field`. I'll add more in the next pr.